Model/Sync/CollectionsProducts/Processor.php

```diff
@@ -292,14 +329,22 @@ private function enrichCollectionsProductsDataWithYotpoIds(array $collectionsPro
         $collectionsProductsProductIds = array_unique($collectionsProductsProductIds);
 
         $enrichedCollectionsProductsEntitiesToSync = [];
-        $collectionsProductsCategoriesIdsToYotpoIdsMap = $this->yotpoCategoryProcessorMain->getYotpoIdsFromCategoriesSyncTableByCategoryIds($collectionsProductsCategoriesIds);
-        $collectionsProductsProductIdToYotpoIdsMap = $this->yotpoProductProcessorMain->getYotpoIdsFromProductsSyncTableByProductIds($collectionsProductsProductIds);
+        $collectionsProductsCategoriesIdsToYotpoIdsMap =
+            $this->yotpoCategoryProcessorMain->getYotpoIdsFromCategoriesSyncTableByCategoryIds(
```

Comment:
Why a new line is needed here?
What is the suggested maximum length of a line?

Reply:
120 is the maximum length